src/Cron/MinutesField.php

```diff
@@ -16,8 +16,14 @@ public function increment(\DateTime $date, $invert = false)
     {
         if ($invert) {
             $date->modify('-1 minute');
+            if($date->format('i')=='59'){
+                $date->modify('+1 hour');
```

Comment:
Perhaps, but with 0 as minute value it seems to work.

Also, I wasn't sure what the implications are as there are core tests failing with that change included.